digdag-tests/src/test/java/utils/TestUtils.java

```diff
@@ -171,12 +185,30 @@ public static CommandStatus main(LocalVersion localVersion, InputStream in, Coll
     }
 
     public static int main(Map<String, String> env, LocalVersion localVersion, Collection<String> args, OutputStream out, OutputStream err, InputStream in)
+    {
+        return main(env, localVersion, args, out, err, in, null);
+    }
+
+    public static int main(
+            Map<String, String> env,
+            LocalVersion localVersion,
+            Collection<String> args,
+            OutputStream out,
+            OutputStream err,
+            InputStream in,
+            RecordableWorkflow.CustomMain.CommandAdder commandAdder)
     {
         try (
                 PrintStream outp = new PrintStream(out, true, "UTF-8");
                 PrintStream errp = new PrintStream(err, true, "UTF-8");
         ) {
-            Main main = new Main(localVersion.getVersion(), env, outp, errp, in);
+            Main main;
+            if (commandAdder != null) {
```

Comment:
Trivial thing, but it it is better to use Optional. (No strong opinion)

Reply:
Ah, I'm used to Kotlin recently and tend to use just a nullable value even in Java. Also, Optional type value in Java isn't null safe, so I may sometimes simply use just nullable value 😄